calabash-cucumber/spec/lib/logging_spec.rb

```diff
@@ -82,4 +82,122 @@
       Calabash::Cucumber.log_info("info")
     end
   end
+
+  describe "file logging" do
+    let(:now) { Time.now }
+
+    it ".timestamp" do
+      expected = now.strftime("%Y-%m-%d_%H-%M-%S")
+      expect(Time).to receive(:now).and_return(now)
+
+      actual = Calabash::Cucumber.send(:timestamp)
```

Comment:
There's going to be one in a million times this fails because the second changes between line 90 and 93 :)